frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -132,6 +132,7 @@ const NameValueEditor_ = withDragDropContext(
                   <Button
                     className="pf-m-link--align-left"
                     data-test-id="pairs-list__add-btn"
```

Comment:
Is the old test ID still used? I think we should either convert other tests to use the new ID or just use the old ID for now instead of having two.

Reply:
removed old test id